app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_status_validation_mixin.py

```diff
@@ -56,3 +58,308 @@ def test_update_experiment_with_invalid_publish_status_error(self):
         )
         self.assertFalse(serializer.is_valid())
         self.assertIn("experiment", serializer.errors)
+
+    @parameterized.expand(
+        [
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.IDLE,
+                True,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.DIRTY,
+                True,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.REVIEW,
+                True,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.APPROVED,
+                False,
+            ],
+            [
+                NimbusExperimentFactory.Lifecycles.CREATED,
+                NimbusExperiment.PublishStatus.WAITING,
+                False,
+            ],
+        ]
+    )
+    def test_update_publish_status_errors_for_create(self, lifecycle, status, valid):
+        experiment = NimbusExperimentFactory.create_with_lifecycle(
+            lifecycle,
+        )
+        serializer = NimbusExperimentSerializer(
+            experiment,
+            data={
+                "publish_status": status,
+                "changelog_message": "test changelog message",
+            },
+            context={"user": self.user},
+        )
+        self.assertEquals(serializer.is_valid(), valid)
+
+    @parameterized.expand(
+        [
+            [
+                NimbusExperimentFactory.Lifecycles.PREVIEW,
```

Comment:
The other thing I'd suggest here is, using the lifecycles is a nice shortcut for when you want something to be in a particular set of statuses, but since we're explicitly testing the combinations of statuses here, the lifecycles are actually hiding which pair of (status, publish_status) we're really talking about, so maybe it would be clearer not to use the lifecycles at all and just explicitly refer to the pair of (status, publish_status)